docs/reference/experimental/async/dataset.md

```diff
@@ -0,0 +1,31 @@
+# Dataset
+
+Contained within this file are experimental interfaces for working with the Synapse Python
+Client. Unless otherwise noted these interfaces are subject to change at any time. Use
+at your own risk.
+
+## API reference
+
+::: synapseclient.models.EntityRef
```

Comment:
Should this be ordered after the Dataset? I imagine most folks will care more about Dataset first than the EntityRef

Reply:
agreed, fixed!